### ISSUE TITLE
Static files forwarding was broken. se2

### ISSUE THREAD

**Issue by duker33** [fidals/stroyprombeton#257]
Launch it on local machine
```
make collectstatic
```

Got it
```
FileNotFoundError: [Errno 2] No such file or directory: '/usr/app/src/front/build'
```

Check if SE has the same problem. If do, fix it too

**Comment by 0pdd** [comment]
@duker33 the puzzle [#266](https://github.com/fidals/stroyprombeton/issues/266) is still not solved.

**Comment by 0pdd** [comment]
@duker33 the puzzle [#274](https://github.com/fidals/stroyprombeton/issues/274) is still not solved; solved: [#266](https://github.com/fidals/stroyprombeton/issues/266).

**Comment by 0pdd** [comment]
@duker33 all 2 puzzles are solved here: [#266](https://github.com/fidals/stroyprombeton/issues/266), [#274](https://github.com/fidals/stroyprombeton/issues/274).
